orc/src/main/java/org/apache/iceberg/orc/ORCSchemaUtil.java

```diff
@@ -41,12 +41,12 @@
  */
 public final class ORCSchemaUtil {
 
-  private enum BinaryType {
+  public enum BinaryType {
```

Comment:
Why is this now public?

Reply:
In the readers and writers, e.g. GenericOrcReader and GenericOrcWriter, I would like to instantiate different converters for fixed v/s binary v/s uuid. Making this public will allow us to reference this enum while parsing the ORC type attribute in those classes. I think it might be cleaner to use a visitor similar to AvroSchemaWithTypeVisitor in GenericOrcReader/Writer so that we don't need to reference these classes there as we get enough information from iceberg types to disambiguate. But the ORC implementation today lacks such visitors.